hudi-common/src/main/java/org/apache/hudi/common/config/DFSPropertiesConfiguration.java

```diff
@@ -58,7 +61,8 @@
   // props read from hudi-defaults.conf
   private static TypedProperties GLOBAL_PROPS = loadGlobalProps();
 
-  private final FileSystem fs;
+  @Nullable
+  private final Configuration hadoopConfig;
```

Comment:
In general we can keep `SerializableConfiguration` that way even if it serializes from driver to executor, it may be ok. But I think we don't do this, otherwise we ll face errros with `fs` to begin with